samples/server/petstore/kotlin-server/ktor/src/main/kotlin/org/openapitools/server/apis/PetApi.kt

```diff
@@ -1,80 +1,80 @@
-        
-        if (principal == null) {
-            call.respond(HttpStatusCode.Unauthorized)
-        } else {
-            call.respond(HttpStatusCode.NotImplemented)
-        }
-    }
-
-
-    get<Paths.findPetsByStatus> {  _: Paths.findPetsByStatus ->
-        val principal = call.authentication.principal<OAuthAccessTokenResponse>()
-        
-        if (principal == null) {
-            call.respond(HttpStatusCode.Unauthorized)
-        } else {
-            val exampleContentType = "application/json"
+/**
+* OpenAPI Petstore
+* This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+*
+* The version of the OpenAPI document: 1.0.0
+* 
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+package org.openapitools.server.apis
+
+import com.google.gson.Gson
+import io.ktor.application.call
+import io.ktor.auth.UserIdPrincipal
+import io.ktor.auth.authentication
+import io.ktor.auth.authenticate
+import io.ktor.auth.OAuthAccessTokenResponse
+import io.ktor.auth.OAuthServerSettings
+import io.ktor.http.ContentType
+import io.ktor.http.HttpStatusCode
+import io.ktor.locations.KtorExperimentalLocationsAPI
+import io.ktor.locations.delete
+import io.ktor.locations.get
+import io.ktor.response.respond
+import io.ktor.response.respondText
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import io.ktor.routing.put
+import io.ktor.routing.route
+
+import org.openapitools.server.Paths
+import org.openapitools.server.infrastructure.ApiPrincipal
+
+
+import org.openapitools.server.models.ApiResponse
+import org.openapitools.server.models.Pet
+
+@KtorExperimentalLocationsAPI
+fun Route.PetApi() {
+    val gson = Gson()
+    val empty = mutableMapOf<String, Any?>()
+
+    route("/pet") {
+            authenticate("petstore_auth") {
+        post {
+            val principal = call.authentication.principal<OAuthAccessTokenResponse>()
+            
+            if (principal == null) {
+                call.respond(HttpStatusCode.Unauthorized)
+            } else {
+                call.respond(HttpStatusCode.NotImplemented)
+            }

+        }
+            }
+    }
+
+
+    delete<Paths.deletePet> {  _: Paths.deletePet ->
+        val principal = call.authentication.principal<OAuthAccessTokenResponse>()
+        
+        if (principal == null) {
+            call.respond(HttpStatusCode.Unauthorized)
+        } else {
+            call.respond(HttpStatusCode.NotImplemented)
+        }

+    }
+
+
+    get<Paths.findPetsByStatus> {  _: Paths.findPetsByStatus ->
+        val principal = call.authentication.principal<OAuthAccessTokenResponse>()
+        
+        if (principal == null) {
+            call.respond(HttpStatusCode.Unauthorized)
+        } else {
+            val exampleContentType = "application/json"

```

Comment:
Why was this file changed?

Reply:
required in the PR checklist runnning:
./mvnw clean package 
./bin/generate-samples.sh
./bin/utils/export_docs_generators.sh

Changed the files